### PR TITLE
Gerrit hook for CLA manager

### PR DESCRIPTION
**Pull request by zdw**
Progress on #7

### CODE REVIEW

File: gerrit/ref-update [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+import argparse
+import json
+import logging
+import re
+import sys
+import urllib.request
+
+# Edit this variable with the URL of the CLA manager instance
+CLA_MANAGER_URL = "https://cla-manager.example/check-cla"
```
Comment (teone) [inline gerrit/ref-update]: should we source this from an environment variable?

**Comment by zdw** [comment]
@hwchiu Could you make sure my CircleCI changes are valid?

File: gerrit/ref-update (hunk 1 above)
Comment (zdw) [reply gerrit/ref-update]: This is run by Gerrit as a hook, which doesn't (to my knowledge) have the ability to set environmental variables.

File: .circleci/config.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ defaults: &defaults
     - image: opennetworking/clam-ci
 
 jobs:
-  test:
+  test-node:
```
Comment (hwchiu) [inline .circleci/config.yml]: Please also change the `workflow` section, should use the `test-node` and `test-py`.
We can add those two `testing jobs` as a requirement for `build job`. it means we will run build only if these two testing jobs return success.

Comment (zdw) [reply .circleci/config.yml]: Made  in new revision

**Pull request merged**